bin/deploy_automation/make_release_checklist.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR/_global_vars.sh"
+source "$SCRIPT_DIR/_shared_functions.sh"
+
+# Compare staging to prod and create a draft pull request in github
+# with commits that are only in staging
+main() {
+  git fetch --all
+
+  # make sure release checklist doesn't exist yet
+  _assert_no_release_checklist
+
+  declare version; version="$(_get_latest_version "$STAGING_BRANCH")"
+  __make_release_checklist "$STAGING_BRANCH" "$PROD_BRANCH" "[RELEASE_CHECKLIST] - v${version}"
+}
+
+__make_release_checklist() {
+  declare source_branch="$1"
+  declare target_branch="$2"
+  declare title="$3"
+
+  declare source_commit; source_commit=$(_get_latest_commit "origin/$source_branch")
+  declare target_commit; target_commit=$(_get_latest_commit "origin/$target_branch")
+
+  declare release_checklist_body; release_checklist_body=$(
+    echo "# Release checklist"
+    echo "_Please check your commits after testing:_"
+    git log --date-order --pretty=format:$'### %an\t* [ ] %h - % %s %d (%cD)' "${target_commit}..${source_commit}" \
+      | awk -F $'\t' '{ a[$1] = a[$1] "\n" $2; } END { for (i in a) print i a[i]; }'
+    echo
+    echo "_(included commits: ($source_commit)..($target_commit), created on $(date '+%Y-%m-%d %H:%M:%S'))_"
+    echo
+    echo "# Release/Hot fixes"
+    echo '_Make sure all your release/hot fixes are listed here. Check your commits after testing and cherry-picking to staging/master accordingly:_'
+  )
+
+  _log "Creating release checklist..."
+  declare response_json
+  if response_json=$(
+    http --ignore-stdin --check-status --timeout=30 POST "${GITHUB_REPOSITORY_API}/pulls" \
+      "Accept:application/vnd.github.shadow-cat-preview+json" \
```

Comment:
maybe add a comment about `shadow-cat-preview`? sounds like this API may change in the future